greeter/README.md

```diff
@@ -121,7 +126,32 @@ export interface Message {
 }
 ```
 
-### Step 3: Create a Greeting Controller
+### Step 3: Create a Simple Greeter Service Class
+
+We're going to create a `SimpleGreetingService` class. In the `src` folder, create a file called `simple.greeting.service.ts` with the following content.
+
+```ts
+export class SimpleGreetingService {
+    
+    async greet(language: string, name: string): Promise<string> {
+      
+      let greeting:string='';  
+
+      switch(language) {
```

Comment:
Hmm, do we want to support language based message in the mock class? I think simply print out a string is good enough.